### PR TITLE
feat: Add runtime to TV, watchTime and watchCount statistic and rename tvEpisodes to seasonEpisodes

### PR DESCRIPTION
**Pull request by JakWai01**
Closes https://github.com/alphahorizonio/consumat.io/issues/111

### CODE REVIEW

File: consumatio/usecases/watch_time.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from consumatio.external.models import *
+from sqlalchemy import text
+
+
+class WatchTime:
+    def get_watch_time(self, tmdb, user, type):
```
Comment (dentremor) [inline consumatio/usecases/watch_time.py:6]: The type annotations are missing here

File: consumatio/usecases/watch_count.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+from consumatio.external.models import *
+from sqlalchemy import text
+
+
+class WatchCount:
+    def get_watch_count(self, tmdb, user, type):
```
Comment (dentremor) [inline consumatio/usecases/watch_count.py:6]: The type annotations are missing here

File: consumatio/usecases/watch_count.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        count = 0
+        if type in "MovieEpisodeTVSeason":
```
Comment (dentremor) [inline consumatio/usecases/watch_count.py:8]: You could use the InvalidParameter exception here

File: consumatio/usecases/watch_time.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        runtime = 0
+        if type == "Movie":
```
Comment (dentremor) [inline consumatio/usecases/watch_time.py:8]: You could check for an invalid parameter and raise an InvalidParameter exception if necessary

**Review by dentremor — CHANGES_REQUESTED** [review]
Please look at the suggested changes. :)

**Comment by JakWai01** [comment]
I implemented most of the requests. I can't implement the invalidArgumentException in watchCount. Furthermore, be aware that most changes that do not affect the basic functionality of the PR will be added in the refactoring later this week.

**Review by dentremor — APPROVED** [review]
LGTM

**Pull request merged**